app/components/UI/Rewards/components/Tabs/LevelsTab/UpcomingRewards.tsx

```diff
@@ -275,7 +275,7 @@ const UpcomingRewards: React.FC = () => {
   };
 
   return (
-    <Box twClassName="py-4 gap-4">
+    <Box twClassName="py-4 px-4 gap-4">
```

Comment:
nit: could use shorthand here

```suggestion
    <Box twClassName="p-4 gap-4">
```